src/styles/counter-theme.css

```
@@ -0,0 +1,67 @@
+  --accent-foreground: 40 40% 99%;
+
+  --destructive: 0 62% 38%;
+  --destructive-foreground: 40 40% 99%;
+
+  --border: 34 20% 80%;
+  --input: 34 20% 80%;
+  --ring: 16 62% 40%;
+
+  --radius: 0.5rem;
+
+  --font-display: 'Zilla Slab', Georgia, serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)